assets/wizards/popups/views/segmentation/SingleSegment.js

```diff
@@ -130,17 +146,29 @@ const SegmentsList = ( { segmentId, wizardApiFetch } ) => {
 					description={ __( '(if using WooCommerce checkout)', 'newspack' ) }
 				>
 					<CheckboxControl
-						checked={ is_donor }
-						onChange={ setIsDonor }
+						checked={ segmentConfig.is_donor }
+						onChange={ updateSegmentConfig( 'is_donor' ) }
 						label={ __( 'Has donated', 'newspack' ) }
 					/>
 					<CheckboxControl
-						checked={ is_not_donor }
-						onChange={ setIsNotDonor }
+						checked={ segmentConfig.is_not_donor }
+						onChange={ updateSegmentConfig( 'is_not_donor' ) }
 						label={ __( "Hasn't donated", 'newspack' ) }
 					/>
 				</SegmentSettingSection>
 			</div>
+
+			{ reach.total > 0 && (
+				<div style={ { opacity: isFetchingReach ? 0.5 : 1 } }>
+					{ __( 'This segment would reach', 'newspack' ) }{' '}
+					<b>
+						{ Math.round( ( reach.in_segment * 100 ) / reach.total ) }
```

Comment:
Could we show the actual segment size as well as the percentage of all users?

Reply:
I was wondering about that, but I think it might be misleading – the actual audience size will be different, so this is just an approximation. For "real" numbers the users should go to GA. WDYT?

Reply:
Why would the actual audience size be different?

Reply:
Only last 30 days worth of data is kept – based on that we can assume the percentage (e.g. 10% of them were subscribers, so it's likely that 10% of _all_ readers are), but when presenting an "actual segment size", we'd have mention that it's just based on last 30 days' visits. By "real" numbers I meant that whole record, not just last 30 days.

Reply:
Ok, not a big deal. We can revisit if anyone questions it.